core/domain/translation_domain_test.py

```diff
@@ -0,0 +1,107 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for translate services."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.domain import translation_domain
+from core.tests import test_utils
+import utils
+
+
+class MachineTranslatedTextTests(test_utils.GenericTestBase):
+    """Tests for the MachineTranslatedText domain object."""
+
+    translation = None
+
+    def setup(self):
```

Comment:
set**U**p*

Reply:
Thank you! I've been wracking my brain over that one. Fixed.